Core/HLE/sceNetAdhoc.cpp

```diff
@@ -259,8 +259,14 @@ static int sceNetAdhocPdpCreate(const char *mac, u32 port, int bufferSize, u32 u
 					// Binding Information for local Port
 					sockaddr_in addr;
 					addr.sin_family = AF_INET;
-					addr.sin_addr.s_addr = INADDR_ANY;
-
+					if (g_Config.bBindLocal)
```

Comment:
Well, `INADDR_ANY` isn't a value for `sin_addr`, but for `sin_addr.s_addr`.  That said, it should still be possible.  It would look fairly ugly though:

```c++
addr_sin_addr.s_addr = g_Config.bBindLocal ? ((sockaddr_in *)&localIP)->sin_addr.s_addr : INADDR_ANY;
```

Personally, I use ternaries where it's simple and clear, often clearer.  In this case, I think it only makes it more complex.

That being said, the convention within PPSSPP is to use 1TBS curlies, rather than Allman as below.  Also, I might just make localIP either a union or sockaddr_in itself, to avoid the casting - would be a lot less ugly.  Not a big deal, though.

-[Unknown]